### ISSUE TITLE
[BUG]: clients() does not record all clients

### ISSUE THREAD

**Issue by lbo34** [datawookie/clockify#13]
### Description of the bug

I am using the clockify package version 0.0.10
In my company, we are using clockify. So far, we have 60 clients. When I am trying to retrieve the list of clients, I only get a dataframe with 50 clients (seems to be in alphabetical order).

### Steps To Reproduce

I use the instruction

clients = clients(concise = F)

**Comment by lbo34** [comment]
I have a similar behaviour with version 0.0.14

**Comment by datawookie** [comment]
Hi @lbo34, aha! Aren't you lucky to have 50 clients. We are a small company with half that number so have not encountered this issue. I think that the results must be paginated. I'll take a look but I'm going to have to ask you to help me with some testing! Thanks, Andrew.

**Comment by datawookie** [comment]
Hi @lbo34, I have implemented a fix. Would you test for me please?

You can install from GitHub as follows:

```
devtools::install_github("datawookie/clockify@paginated-clients")
```

Let me know how that goes.

Thanks, Andrew.

**Comment by lbo34** [comment]
It works! Thanks!!!!

**Comment by datawookie** [comment]
Aha! Wonderful. Okay, let me merge that into `master` and make a new release. One moment...

**Comment by datawookie** [comment]
Fixed in d14881897c44c4174bf90a930709ba5eeff37d5d.

@lbo34 please install latest version on the `master` branch (0.0.15). I'm busy adding quite a few new features and will publish on CRAN when I am done. Thanks very much for bringing this issue to my attention.

**Comment by lbo34** [comment]
Done. It's working now. Thanks. 
I'm looking forward to see the new features!
Thanks
